float64[] stamps
float64[] values
---
geometry_msgs/TwistStamped[] twist_stamped_vec
